health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignDetails.js

```
@@ -255,17 +312,34 @@
+                  icon="CheckCircleOutline"
+                  label={t("HCM_CREATE_CAMPAIGN")}
+                  onClick={onsubmit}
+                  isDisabled={
+                    campaignData?.boundaries?.length === 0 ||
+                    campaignData?.deliveryRules?.length === 0 ||
+                    campaignData?.resources?.length === 0 ||
+                    modulesData?.length === 0
+                  }
+                  type="button"
+                  variation="primary"
+                />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)